src/sass/app_2/_common/base/themes.scss

```diff
@@ -64,6 +66,8 @@ $themes: (
         text_primary_color               : $COLOR_DARK_BLUE_7,
         text_secondary_color             : $COLOR_DARK_BLUE_7,
         won_gradient_color               : $COLOR_DARK_GREEN_GRADIENT,
+        wip_btn_color                    : $COLOR_DARK_BLUE_2,
+        wip_color                        : $COLOR_WHITE,
```

Comment:
we can also use themed('icon_color') for the transition from $COLOR_DARK_BLUE_5 to $COLOR_WHITE

Reply:
@cakasuma , i think it's best to keep it separate even if the color changes are the same, just in case in the future if we need to update the colors for this specific element or if we add more themes with different color transitions. Additionally, using `icon_color` for an element that's not actually an `icon` is a bit confusing.